.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: bash
+services: docker
+
+install:
+  - git clone https://github.com/docker-library/official-images.git ~/official-images
+
+script:
+  - docker build -t "dopplerrelay/doppler-relay-akamai-publish" .
```

Comment:
Could you also add a link in Readme file to CI and Docker Hub?

Reply:
Ready!